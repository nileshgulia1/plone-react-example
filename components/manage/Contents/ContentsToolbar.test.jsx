import React from 'react';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-intl-redux';

import ContentsToolbar from './ContentsToolbar';

const mockStore = configureStore();

describe('ContentsToolbar', () => {
  it('renders a contents toolbar component', () => {
    const store = mockStore({
      intl: {
        locale: 'en',
        messages: {},
      },
    });
    const component = renderer.create(
      <Provider store={store}>
        <ContentsToolbar location={{ pathname: '/blog' }} />
      </Provider>,
    );
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
