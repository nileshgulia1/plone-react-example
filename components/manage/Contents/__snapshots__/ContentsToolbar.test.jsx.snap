// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentsToolbar renders a contents toolbar component 1`] = `
<div>
  <a
    className="item"
    id="toolbar-cancel"
    onClick={[Function]}
    style={Object {}}
  >
    <i
      aria-hidden="true"
      className="red remove big icon"
      title="Cancel"
    />
  </a>
</div>
`;
